packages/controls-navigator/src/NavigatorControl.vue

```diff
@@ -0,0 +1,293 @@
+<template>
+  <div :class="getNavigateClass()">
+    <v-stepper :model-value="currentStep - 1">
+      <template v-for="index in steps" :key="index">
+        <v-stepper-item
+          :editable="navigationType === 'tab' || currentStep > index"
+          :title="prompts[index - 1] ? prompts[index - 1].name : ''"
+          :complete="getComplete(index)"
+          :class="getStepClass(currentStep, index)"
+          @click="gotoStep(index)"
+        />
+        <div class="breadcrumbs-container">
+          <YOUIBreadcrumbs
+            :id="`breadcrumbs-${index}`"
+            class="breadcrumbs"
+            :breadcrumbs="answers[prompts[index - 1] ? prompts[index - 1].name : undefined]"
+          />
+        </div>
+      </template>
+    </v-stepper>
+  </div>
+</template>
+
+<script setup>
+import { computed, ref, toRefs, unref, watchEffect } from "vue";
+import YOUIBreadcrumbs from "./YOUIBreadcrumbs.vue";
+import * as _ from "lodash";
```

Comment:
You can bring only isEmpty instead of all lodash

Reply:
has changed it.